### PR TITLE
Feature#10login design

### LINKED ISSUE #10: Login's design
# REQUIREMENTS
- Create the login stylesheet
- Apply the styles to login html 

# DESIGN
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Follow the login mockup

### PR DESCRIPTION
**Pull request by AngelBFdev**
# Description
- Added global styles
- Added variables
- Added login Stylesheet
 
  ## Screenshots 
Mockup
image
Frontend
image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import "bootstrap/scss/bootstrap";
+@import "./base/variables.scss";
+
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss]: Delete tihis extra line

File: app/assets/stylesheets/components/_login.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        display: grid;
+        font-family: $poppins;
+        gap: 2rem;
+        grid-template-columns: 1fr 1fr;
+        margin: 0 auto;
+        max-width: 50rem;
+        .login{
+            background-color: $modal_color;
+            border-radius: 5%;
+            padding:1rem 0rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_login.scss]: add a space between each block

File: app/assets/stylesheets/components/_login.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            padding: 1rem;
+            div{
+                p, h1{
+                    margin-bottom: 2rem;
+                    text-align: left;
+                }
+            }
+        }
+    }
+}
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_login.scss]: Review the identation, the ideal is used 2 spaces not 4

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/application.scss]: Done

File: app/assets/stylesheets/components/_login.scss (hunk 2 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_login.scss]: Done

File: app/assets/stylesheets/components/_login.scss (hunk 3 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_login.scss]: Done

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by DerekAyala — APPROVED** [review]

**Review by kevinnio — APPROVED** [review]

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -12,21 +12,18 @@
...
-      <%= link_to "Logout", destroy_user_session_path, method: :delete, :class => 'navbar-link'  %>
-    <% else %>
-        <%= link_to "Sign up", new_user_registration_path, :class => 'navbar-link'  %> |
-        <%= link_to "Login", new_user_session_path, :class => 'navbar-link'  %>
+      <nav class="navbar navbar-light bg-light">
+        <li class="nav-item active">
+          <a class="nav-link" href="/feed">Seeker</a>
+            Logged in as <strong><%= current_user.email %></strong>.
+          <%= link_to 'Edit profile', edit_user_registration_path, :class => 'navbar-link' %> |
+          <%= link_to "Logout", destroy_user_session_path, method: :delete, :class => 'navbar-link'  %>
```
Comment (YilKanda) [inline app/views/layouts/application.html.erb]: Use the most current convention.

```suggestion
          <%= link_to 'Edit profile', edit_user_registration_path, class: 'navbar-link' %> |
          <%= link_to "Logout", destroy_user_session_path, method: :delete, class: 'navbar-link'  %>
```

**Review by YilKanda — APPROVED** [review]
LGTM! Just remember to use the current Rails code convention.

**Pull request merged**